doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/proc-guided-decision-tables-columns-sort.adoc

```diff
@@ -0,0 +1,11 @@
+[id='proc-guided-decision-tables-columns-sort_{context}']
+= Sorting column values in guided decision tables
+
+You can sort the values in columns that you created in a guided decision table.
```

Comment:
I think this, and the "you" in prereqs, does not account for scenarios when someone else created the column. But up to Heena, I'd say.

Reply:
I think I'd keep you here since we're following a flow.